### DISCUSSION TITLE
Translations setup on Crowdin

### DISCUSSION THREAD

**Discussion by guustnieuwenhuis** [MasaCMS/MasaCMS#33] [Announcements]
Do you speak a language other than English and you're willing to help out translating or proofreading? You're welcome to help us out!

We've setup a Crowdin project to collaborate together on translating the Masa CMS interface into as much languages as possible.: https://translations.masacms.com

**Comment by JonasEriksson** [comment]
Will get on the German one! Not sure if a Swedish one is required, most will happily use the English one in Sweden/Norway/Denmark methinks.

**Reply by comradekingu** [reply]
I could help with that, but I have stopped using Crowdin since their terms and conditions are so egregious.
Hoping to see this project move to Weblate.

**Reply by guustnieuwenhuis** [reply]
@comradekingu We've reviewed the R&C and PP, they look pretty standard to us.

**Reply by jochemd** [reply]
Could you elaborate on why their T&C are so egregious? I see some issues with their privacy policy, but the T&C are pretty standard.

And obviously some people using Crowdin would not stop you from just creating a pull request.

**Reply by comradekingu** [reply]
Crowdin can't be used, studied, changed or distributed freely.
As an only-as-a-service offering, and as such their libre software hosting terms are comparitivly only for non-commercial projects.

Reading the Crowdin ToS is meaningless without reading the others (including [the cookie policy](https://support.crowdin.com/cookies/) hidden behind "+4 more"
![bilde](https://user-images.githubusercontent.com/13802408/145426550-ccfbbe3d-e437-4250-b043-9b08997a3a18.png)
and then covered by
![bilde](https://user-images.githubusercontent.com/13802408/145426415-42d0821f-2389-4705-809b-c2d6c0d6a25c.png)
, and the respective documents for services they use) because it it all is seemingly written to appear harmless.
For every thing that is detailed, it opens up a possibility that is exploited somewhere else, and differing justification is used in the same sentences. Standard fare, but far from fair.

Reading the Crowdin ToS is meaningless unless you do before every use, and every month

>Your continued use of the Crowdin Services after such changes have been posted as provided above constitutes your binding acceptance of such changes. Such amended Terms will automatically be effective upon the earlier of (i) your continued use of the Crowdin Services, or (ii) 30 days from posting of such modified Terms on or through the Platform. Notwithstanding the foregoing, the resolution of any dispute that arises between you and Supplier will be governed by the Terms in effect at the time such dispute arose.

I could send remote PRs, if that worked, and if it didn't commission my work into their closed source translation memory and send my data to their partners. Before we get into the "where does it say that".

I'll give you loophole one:

>If Client has designated Users and granted them Authorization, such Users will be deemed to be authorized to act on behalf of Client when using the Account.

And before we get to "oh, but it could be that this merely means _use_ of the account"

>To accept these Terms for itself or on behalf of a Client, a person must have the legal capacity to do so. In the case of an individual, the individual must be at least 18 years of age or have valid authorization from his/her legal representative or custodian.

I hope to find you agree that is egregious, and don't intend to do this.

---

From a translator perspective, the actual usability is not there, and no quality translations are facilitated as a result.
Reason one is the lack of overview, inability to see more languages at once, and a voting system that keeps any level of consistency from being applied.

Your CLA is a DCO, and it wouldn't be needed if the license was stated, or the service had thought to implement it.
6.4 https://hosted.weblate.org/legal/terms/
